src/style.css

```diff
@@ -0,0 +1,55 @@
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.img {
+  position: absolute;
+  top: 45px;
+  left: 35px;
+}
+
+.image-small {
+  width: 11px;
+  top: 124px;
+  left: 23px;
+}
+
+.input {
+  display: flex;
+  outline: none;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  margin: 20px auto;
+  padding-left: 62px;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+}
+
+.input-small {
+  height: 42px;
+  font-size: 14px;
+  font-weight: 400px;
+  padding: 12px 0 14px 33px;
+  background-size: 11px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
font-family: inherit;

Reply:
No, it doesn`t work with inherit.
Fonts became bolder as it used to be..